vidarrtest-regression.json.in

```diff
@@ -361,7 +362,7 @@
     "description": "ichorCNA workflow test, using one bam as input",
     "arguments": {
       "ichorCNA.bamQC.bamQCMetrics_refFasta": "/.mounts/labs/gsi/modulator/sw/data/hg38-p12/hg38_random.fa",
-      "ichorCNA.bamQC.bamQCMetrics_refSizesBed": "/.mounts/labs/gsiprojects/gsi/gsiusers/blujantoro/tools-resources/interval-files/standard/ALL.AS,CH,CM,NN,SW,WG.hg38.bed",
+      "ichorCNA.bamQC.bamQCMetrics_refSizesBed": "/.mounts/labs/gsiprojects/gsi/gsiusers/blujantoro/tools-resources/interval-files/standard/ALL.AS,CH,CM,NN,PG,SW,WG.hg38.bed",
```

Comment:
OH! great catch! this must've been breaking things. This is the new version of the file in the interval-files repo, maybe we should be getting this from a more standard location.